source/module_parameter/parameter.h

```diff
@@ -38,6 +41,17 @@ class Parameter
     Input_para input;
     // System parameters
     System_para sys;
+
+#ifdef __EXX
+public:
+    const Exx_Info& exx_info = exx_info_;
+    void set_exx_info() { this->exx_info_.set(this->inp); }
+    void set_exx_abfs_Lmax(const int l) { this->exx_info_.info_ri.abfs_Lmax = l; }
+    std::vector<std::string>& get_exx_abfs_file() { return this->exx_info_.info_ri.files_abfs; }
+    void set_exx_postscf();
+private:
+    Exx_Info exx_info_;
```

Comment:
Could you create a new struct to wrap up variables such as 'exx' from 'Input_Para'? Maybe call it 'Exx_Para'. Types of some variables can be directly refactored and adjusted. For example: "string exx_hybrid_alpha" can be changed to "double exx_hybrid_alpha" directly. Now 'exx_info' is too complex, it can be placed in 'esolver', initialized through 'Exx_Para'.